src/HotChocolate/Core/src/Types.Scalars/NegativeIntType.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using HotChocolate.Language;
+using HotChocolate.Types.Scalars;
+
+namespace HotChocolate.Types
+{
+    /// <summary>
+    /// The NegativeIntType scalar type represents Integers that will have a value less than 0.
+    /// </summary>
+    public class NegativeIntType : IntType
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NegativeIntType "/> class.
+        /// </summary>
+        public NegativeIntType()
+            : this(
+                WellKnownScalarTypes.NegativeInt,
+                ScalarResources.NegativeIntType_Description)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NegativeIntType "/> class.
+        /// </summary>
+        public NegativeIntType(
+            NameString name,
+            string? description = null,
+            BindingBehavior bind = BindingBehavior.Explicit)
+            : base(name, description, int.MinValue, -1, bind)
+        {
+            Description = description;
+        }
+
+        /// <inheritdoc />
+        protected override bool IsInstanceOfType(int runtimeValue)
+        {
+            return runtimeValue < 0;
```

Comment:
You have access to the protected `MaxValue` :) 
```suggestion
            return runtimeValue <= MaxValue;
```